build/wix/mixxx.wxs

```diff
@@ -218,11 +218,11 @@
     -->
 
     <!-- minimum OS version check disabled for now because we do not want to enforce installing mixxx on older systems.
-         Anyway, we display a warning message when installing on XP and below
+         Anyway, we display a warning message when installing on XP and below -->
     <Condition Message="!(loc.OS2Old)">
        <![CDATA[Installed OR (VersionNT >= 501)]]>
```

Comment:
change this too to 600 instead of 501.

Also, I just realized that we have to change the "OS2Old" message and translations in wix/Localization.

Reply:
Ok I'll update it